docs/source/api/v2/servers_id_deliveryservices.rst

```diff
@@ -239,3 +239,81 @@ Response Structure
 
 
 .. [#tenancy] Only the :term:`Delivery Services` visible to the requesting user's :term:`Tenant` will appear, regardless of their :term:`Role` or the :term:`Delivery Services`' actual 'server assignment' status.
+
+``POST``
+========
+Assign an arbitrary number of delivery services to a single server.
```

Comment:
Doesn't need to be linked again, but should be "Delivery Services" not "delivery services"

Reply:
Changed to Delivery Services in cfbcfe347a.